src/styles/blocks/card.scss

```diff
@@ -0,0 +1,87 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+
+  width: 200px;
+  padding: 32px 16px 0;
+
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    max-width: 160px;
+    height: 134px;
+    align-self: center;
+  }
+
+  &__name {
+    margin-top: 40px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: #060b35;
+  }
+
+  &__code {
+    margin-top: 4px;
+    font-size: 10px;
+    line-height: 14px;
+
+    color: #616070;
+  }
+
+  &__review {
+    margin-top: 16px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    &-count {
+      font-size: 10px;
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 24px;
+    align-items: center;
+    line-height: 18px;
+
+    &-name {
+      color: #616070;
+      font-size: 12px;
+    }
+
+    &-price {
+      font-size: 16px;
+      font-weight: 700;
+      color: #060b35;
+    }
+  }
+
+  &__button {
+    margin: 16px 0;
+    height: 40px;
+
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+
+    border-radius: 5px;
+    border: none;
+    background-color: #00acdc;
+    color: #fff;
+
+    &:hover {
+      background-color: #fff;
+      color: #00acdc;
+      border: 1px solid #00acdc;
+      transition: all 0.5s;
```

Comment:
It is better not to set transitions for everything. It's correct to indicate specifically.